File: YoonJeong/017주차/주식가격.py
def solution(prices):
    answer = []
    
    for i in range(len(prices)) :
        for j in range(i, len(prices)) :
            if prices[j] < prices[i] :
                answer.append(j - i)
                break
        else :
            answer.append(len(prices) - i - 1)

    return answer

File: juhee2/17주차/주식가격.java
class Solution {
    public int[] solution(int[] prices) {
        int[] answer = new int[prices.length];
        
        for (int i = 0; i < prices.length; i++) {
            int time = 0;
            for (int j = i+1; j < prices.length; j++) {
                time++;
    if(prices[i] > prices[j]) {
        break;
}
                
}
            answer[i] = time;
}
        return answer;
    }
}
